Makefile

```diff
@@ -6,6 +6,10 @@ build:
 workspace:
 	dune build --workspace=dune-workspace.template
 
+workspace-ocamlmerlin:
+	dune build --workspace=dune-workspace.template \
+		src/frontend/ocamlmerlin_server.exe
```

Comment:
This feels out of place in this PR. Also, I'd suggest building "merlin.install" instead of "src/frontend/ocamlmerlin_server.exe").

Reply:
Right.